keystore/src/connection/platform/generic/migrations/V12__unique_id_constraints.sql

```diff
@@ -10,4 +10,17 @@ SELECT lower(sha256_blob(pk)), pk, sk FROM mls_encryption_keypairs;
 
 DROP TABLE mls_encryption_keypairs;
 
-ALTER TABLE mls_encryption_keypairs_new RENAME TO mls_encryption_keypairs;
\ No newline at end of file
+ALTER TABLE mls_encryption_keypairs_new RENAME TO mls_encryption_keypairs;
+
+---- mls_epoch_encryption_keypairs ----
+CREATE TABLE mls_epoch_encryption_keypairs_new (
+    id_hex TEXT UNIQUE,
+    keypairs BLOB
+);
+
+INSERT INTO mls_epoch_encryption_keypairs_new (id_hex, keypairs)
+SELECT lower(hex(id)), keypairs FROM mls_epoch_encryption_keypairs;
```

Comment:
Same question for `lower` as before.

Reply:
SQLite's `hex()` always returns upper case, and rust's always returns lower case. So I decided to do the conversion here.